federatedscope/core/fed_runner.py

```diff
@@ -30,6 +30,10 @@ def __init__(self,
         if self.mode == 'standalone':
             self.shared_comm_queue = deque()
             self._setup_for_standalone()
+            # in standalone mode, by default, we print the trainer info only once for better logs readability
+            trainer_representative = self.client[1].trainer
```

Comment:
why is `client[1]`? any similar concept like "chief_worker" in ps-worker paradigm?

Reply:
Here the "[1]" just indicates the first client since we use at least one clients in all cases. 

Arguably, we can modify the `trainer_representative ` access accordingly after we introducing a similar concept like "chief_worker" in future PR, which may be required by asynchronous simulation.